src/averageAges.js

```diff
@@ -15,11 +15,16 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  let men = people.filter((x) => (x.sex === 'm'));
+
+  if (century !== undefined) {
```

Comment:
> replace `if ()` statement with &&, || or ?:

use ternary and check it inside the first `filter` callback to define your condition inside
fix everywhere